types/tpdirect/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for non-npm package tpdirect-browser 1.0
+// Project: https://github.com/Yu-Jack/DefinitelyTyped
```

Comment:
```suggestion
// Project: https://docs.tappaysdk.com/tutorial/en/home.html
```
AFAIK this should point to project home page, not the DT fork

Reply:
Okay, I'll fix! Thank you!